demosplan/DemosPlanCoreBundle/Repository/ProcedurePhaseRepository.php

```diff
@@ -12,8 +12,20 @@
 
 namespace demosplan\DemosPlanCoreBundle\Repository;
 
+use DemosEurope\DemosplanAddon\Contracts\Entities\ProcedureInterface;
+use DemosEurope\DemosplanAddon\Contracts\Repositories\ProcedurePhaseRepositoryInterface;
 use DemosEurope\DemosplanAddon\Logic\ApiRequest\FluentRepository;
 
-class ProcedurePhaseRepository extends FluentRepository
+class ProcedurePhaseRepository extends FluentRepository implements ProcedurePhaseRepositoryInterface
 {
+    public function getProcedureByInstitutionPhaseId(string $phaseId): ProcedureInterface|null
```

Comment:
Minor: As returnType you can also use ?ProcedureInterface which means the same.

Reply:
thanks [f457005](https://github.com/demos-europe/demosplan-core/pull/2906/commits/f457005d7052a835d08276e9109879f1d940945b)